### ISSUE TITLE
Windows build doesn't work with Clang/LLVM 8.0.0

### ISSUE THREAD

**Issue by jxyang** [openenclave/openenclave#1801]
With the latest release of Clang/LLVM 8.0.0, openenclave fails to build on Windows with the following error message:  

```
[348/1757] Building CXX object 3rdparty\libcxx\CMakeFiles\libcxx.dir\libcxx\src\algorithm.cpp.obj
FAILED: 3rdparty/libcxx/CMakeFiles/libcxx.dir/libcxx/src/algorithm.cpp.obj 
"C:/Program Files/Git/bin/bash.exe" "C:/openenclave/scripts/clangw" "-DLIBCXXRT -DOE_API_VERSION=2 -DOE_BUILD_ENCLAVE -DOE_LIBC_SUPPRESS_DEPRECATIONS -D_LIBCPP_BUILDING_LIBRARY -D_LIBCPP_PROVIDES_DEFAULT_RUNE_TABLE -I..\..\3rdparty\libcxx\libcxx\src -I..\..\3rdparty\libcxx\..\libcxxrt\libcxxrt\src -I..\..\include -Ioutput\include\openenclave\libcxx -Ioutput\include\openenclave\libc /DWIN32 /D_WINDOWS /W3 /GR /EHsc /MDd /Zi /Ob0 /Od /RTC1   -Wall -Werror -Wpointer-arith -Wconversion -Wextra -Wno-missing-field-initializers -fno-strict-aliasing -mxsave -fno-builtin-malloc -fno-builtin-calloc -fno-builtin -mllvm -x86-speculative-load-hardening -Wno-error -m64 -fPIE -nostdinc -fno-stack-protector -fvisibility=hidden -ftls-model=local-exec -nostdinc++ -std:c++14 /showIncludes -o 3rdparty\libcxx\CMakeFiles\libcxx.dir\libcxx\src\algorithm.cpp.obj -c ..\..\3rdparty\libcxx\libcxx\src\algorithm.cpp"
In file included from ..\..\3rdparty\libcxx\libcxx\src\algorithm.cpp:10:
In file included from output\include\openenclave\libcxx\algorithm:641:
output\include\openenclave\libcxx\type_traits:740:56: error: _Float16 is not supported on this target
template <>          struct __libcpp_is_floating_point<_Float16>    : public true_type {};
```
#1652

**Comment by CodeMonkeyLeet** [comment]
@jxyang can you verify that this is no longer an issue and close this out given that #2026 has merged without incident?

**Comment by jxyang** [comment]
This is resolved by updating libcxx to 8.0.
